CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# Data Structures and Algorithms
+## Contributions
+
+### Community Guidelines
+
+Hi there! Welcome to the Data Structures and Algorithms Project created by shree1999! Here you will find the guidelines for how to contribute to this project.
+
+1. Please fork the project into your own repository.
+2. Make a clean branch to work off of
+     - Be specific when naming your branch
+3. Thoroughly terst and validate your additions before making a pull request
```

Comment:
Hi @T19700 a small spelling mistake `terst` should be `test`

Reply:
On The contributors list I think we can do some automation to get the contributor's list and put it in README.md file 
Let me take this up.

Reply:
I think I fixed the spelling error now! Good catch!